src/App/Utilities/constants.js

```diff
@@ -0,0 +1,288 @@
+import React from "react";
+import { spacingClasses, borderClasses, breakpointClasses, hidingElementClasses, flexClasses, imageClasses, sizingClasses, textClasses, visibilityClasses } from "./classes";
+import CodeTagComponent from "@components/CodeTags";
+
+const UtilityPresenter = ({ util, css, example, breakpointVersion }) => (
+    <div className="d-flex justify-content-between mb-2">
+        <CodeTagComponent code={util} type="primary"/>
```

Comment:
in the future we could maybe do like in the DNB Changelog page, where they have a tag on which you can copy the value by clicking on it,
here it could make sense to have it on the classes names?
I mean this for an eventual future iteration 💡👨‍🚀✨🌌🛰🚀☄

Reply:
Good idea :D